tests/integration/list/resources/resources_integ_base.py

```diff
@@ -0,0 +1,45 @@
+import os
+from unittest import TestCase
+from pathlib import Path
+import uuid
+import shutil
+import tempfile
+
+
+class ResourcesIntegBase(TestCase):
```

Comment:
There's a lot of duplication here. I think it would be best to create a `ListIntegBase` base class to house the common logic used by each of the sub-commands that they can all then extend.

Reply:
Created ListIntegBase